### ISSUE TITLE
Beautify merges padding overrite incorrectly (.SCSS)

### ISSUE THREAD

**Issue by Moustachey** [Glavin001/atom-beautify#2324]
# Description

When I beautify I noticed that my 'padding-top' rule was being combined in to the above 'padding' rule, which made both the padding top AND the padding bottom rule to become the new value (when it should have only affected the padding-top value or not touched it at all).

# Input Before Beautification

This is what the code looked like before:

```
.nav-item {
margin: 0 10px !important;
padding: 6px 12px !important;
padding-top: 3px !important;
}
```

# Expected Output

The beautified code should have looked like this:

```
.nav-item {
margin: 0 10px !important;
padding: 3px 12px 6px 12px !important;
}
```

# Actual Output

The beautified code actually looked like this:

```
.nav-item {
margin: 0 10px !important;
padding: 3px 12px !important;
}
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

**Comment by prettydiff** [comment]
I fixed this in the parser just now, but....

I still need to publish the next patch version of the parser when I finish a small fix for some JavaScript defects I recently found.  Then the fix will be automatically available in the latest version of Pretty Diff, but I still need to get the latest version of Pretty Diff into Atom Beautify.  I spent 2018 rewriting the application from scratch in a new language (TypeScript) to execute faster, have smaller code, and have everything in a single file with a simplified API and uniform conventions and so on.
